role-assignments.tf

```diff
@@ -0,0 +1,11 @@
+#module "role-assignements" {
```

Comment:
if no need remove commented code, if you need it for now have TODO: description why you have this code commented